### ISSUE TITLE
GenerateParams stop_sequences is not accepting a list of strings as input: pydantic issue

### ISSUE THREAD

**Issue by mirianfsilva** [IBM/ibm-generative-ai#141]
Remove **min_length** from stop_sequences list schema. 

It was working in version=0.2.2, when added the **min_length** in the schema, it stopped working and accepting stop sequence lists like `["\n", "###", "."] `.

https://github.com/IBM/ibm-generative-ai/blob/f7680ea11e280f39a13ca247fbb1ca3066430f52/src/genai/schemas/generate_params.py#L59

```
params = GenerateParams(
    decoding_method="greedy", max_new_tokens=15, min_new_tokens=1, stream=False, stop_sequences=["\n", "###", "."]
)
```
error: 
```pydantic.error_wrappers.ValidationError: 1 validation error for GenerateParams
stop_sequences -> 0
  ensure this value has at least 1 characters (type=value_error.any_str.min_length; limit_value=1)
```

### If the bug is confirmed, would you be willing to submit a PR?

Yes

**Comment by mtake** [comment]
I have the same issue. For some reason, pydantic doesn't count `\n` as a character. So the length of the first stop sequence in your example `"\n"` is zero, which doesn't meet `min_length=1 `condition.
We can no longer pass `"\n"` to the stop_sequences because it fails the range check. This is quite inconvenient since our benchmark harness HELM uses it in many cases.
Question: Is this an intentional limitation, in other words, isn’t `"\n"` allowed as a stop sequence?

**Comment by Tomas2D** [comment]
The `\n` should be obviously allowed; this mistake will be fixed in the upcoming release.

**Comment by Tomas2D** [comment]
Released in [v0.2.6](https://github.com/IBM/ibm-generative-ai/releases/tag/v0.2.6) 🚀 

Thanks! 🥇
